apps/andi/lib/andi/input_schemas/input_converter.ex

```diff
@@ -266,7 +268,22 @@ defmodule Andi.InputSchemas.InputConverter do
   defp update_context_from_smrt_step(context, _), do: context
 
   defp encode_extract_step_body_as_json(%{body: body} = smrt_extract_step) when body not in ["", nil] do
-    Map.put(smrt_extract_step, :body, Jason.encode!(body))
+    case body do
```

Comment:
Originally, I thought this was the error, but turns out it was overwriting incorrectly later in the PR. I left this here for safety though in case of a malformed message body.